build.gradle

```diff
@@ -34,7 +34,8 @@ def libraries = [
         "org.jetbrains.kotlinx:kotlinx-serialization-json-jvm",
         "org.jetbrains.kotlinx:kotlinx-serialization-cbor-jvm",
         "org.jetbrains.kotlinx:atomicfu-jvm",
-        "org.jetbrains.kotlinx:kotlinx-datetime-jvm"
+        "org.jetbrains.kotlinx:kotlinx-datetime-jvm",
+        "org.jetbrains.kotlinx:kotlinx-io-core-jvm"
```

Comment:
This [appears](https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-io-core-jvm/0.5.1) to depend on `kotlinx-io-bytestring-jvm`

Reply:
should this also be listed in the readme?

Reply:
Yes, I think so. All of the others are.

Reply:
resolved